src/styles/globals.css

```diff
@@ -28,4 +28,19 @@ body[style] {
 
 .min-h-screen-1px {
     @apply min-h-[calc(100vh_-_1px)];
+}
+
+body{
+    @apply dark:bg-black;
+}
+
+/* if dark mode make scrollbar dark */
+@media (prefers-color-scheme: dark) {
+    ::-webkit-scrollbar {
+        @apply bg-black;
+    }
+    ::-webkit-scrollbar-thumb {
+        @apply bg-gray-500;
+        border-radius: 10px;
+    }
 }
```

Comment:
Custom scrollbar styles can be unpredicable on mobile browsers and stuff.

After some [digging](https://stackoverflow.com/a/66097469/8239711) though this should do the trick:
```
:root {
    color-scheme: dark;
}
```